src/Features/LanguageServer/Protocol/Handler/Completion/CompletionHandler.cs

```diff
@@ -62,7 +62,7 @@ static LSP.CompletionItem CreateLSPCompletionItem(LSP.CompletionParams request,
             static TCompletionItem CreateCompletionItem<TCompletionItem>(LSP.CompletionParams request, CompletionItem item) where TCompletionItem : LSP.CompletionItem, new()
                 => new TCompletionItem
                 {
-                    Label = item.DisplayText,
+                    Label = item.DisplayTextPrefix + item.DisplayText + item.DisplayTextSuffix,
                     InsertText = item.Properties.ContainsKey("InsertionText") ? item.Properties["InsertionText"] : item.DisplayText,
```

Comment:
fwiw, this line is bad.  it's examining an internal implementation detail of a completion item.  This could change at any time and would break.